core/vdbe/insn.rs

```diff
@@ -40,102 +40,102 @@ impl CmpInsFlags {
 
 #[derive(Description, Debug)]
 pub enum Insn {
-    // Initialize the program state and jump to the given PC.
+    /// Initialize the program state and jump to the given PC.
```

Comment:
For future reference, please do cleanups in separate PRs that don't do any functional changes. Much easier to review.